composer.json

```
@@ -1,27 +1,48 @@
 {
-    "name": "bitexpert/composer-phive-bridge",
-    "description": "Update tools via phive on composer update",
-    "type": "composer-plugin",
-    "license": "MIT",
-    "authors": [
-        {
-            "name": "Andreas Heigl",
-            "email": "a.heigl@bitexpert.de"
-        }
+  "name": "bitexpert/composer-phive-bridge",
+  "description": "Update tools via phive on composer update",
+  "type": "composer-plugin",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Andreas Heigl",
+      "email": "a.heigl@bitexpert.de"
+    }
+  ],
+  "require": {
+    "php": "^7.4|~8.0.0",
```

Being explicit is helpful.

```suggestion
    "php": "^7.4 || ~8.0.0",
```